.travis.yml

```diff
@@ -16,9 +16,8 @@ matrix:
       sudo: required
       language: python
       python: 3.7
-    # To maximise compatibility pick earliest image, OS X 10.12
     - os: osx
-      osx_image: xcode8.3
+      osx_image: xcode9.3
```

Comment:
Thanks for fixing this issue @dybber!

In retrospect it makes sense having to upgrade this, as PyQt 5.13 needs macOS High Sierrra 10.13, or newer, and this version of xcode is the smalles on 10.13: 
https://docs.travis-ci.com/user/reference/osx/#jdk-and-macos

Do you think it would be useful to add a note here indicating we should match this version with the lowest supported by PyQt?

Reply:
Makes sense to add that, yes. Took quite some trial and error to figure out, would be nice if we didn't have to go through that next time.

Reply:
Will you add the note? I don't know if you have commit access, just seems a lot to create a full new pull request for a small comment